proxies/live/apiproxy/policies/AssignMessage.AddBaseUrlHeader.xml

```diff
@@ -0,0 +1,9 @@
+<AssignMessage async="false" continueOnError="false" enabled="true" name="AssignMessage.AddBaseUrlHeader">
+  <Add>
```

Comment:
The header should use 'set' rather than 'add' declaration. Otherwise the value will be appended to whatever client sends :)